sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestAdministrationClient.java

```diff
@@ -109,7 +104,7 @@ public final class LoadTestAdministrationClient {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<BinaryData> createOrUpdateServerMetricsConfigWithResponse(String testId, BinaryData body,
         RequestOptions requestOptions) {
-        return this.serviceClient.createOrUpdateServerMetricsConfigWithResponse(testId, body, requestOptions);
+        return this.client.createOrUpdateServerMetricsConfigWithResponse(testId, body, requestOptions).block();
```

Comment:
@srnagar why do we have the generation with typespec generating blocking calls?

Reply:
This is not a generated method. It must have been manually written or modified after generation as there is no `@Generated` annotation on this method.

Reply:
I have fixed the commit used in the generation and now sync stack is enabled and we are generating non blocking calls.